recipes/tinycthreadpool/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
Please, follow https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates/cmake_package/all/test_package template, and please, add test_v1_package too. You can copy directly from the template and update.

CONAN_LIBS is almost deprecated, we no longer recommend as it's fragile. Instead, prefer CMake targets, which is safer for link and finding folders.